.github/workflows/release_linux_pytorch_wheels.yml

```diff
@@ -15,14 +15,13 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - { AMDGPU_FAMILIES: gfx94X-dcgpu, PYTORCH_TARGET_ARCH: gfx942 }
-          - { AMDGPU_FAMILIES: gfx110X-dgpu, PYTORCH_TARGET_ARCH: gfx1100 }
-          - { AMDGPU_FAMILIES: gfx120X-all, PYTORCH_TARGET_ARCH: gfx1201 }
+          - { AMDGPU_FAMILIES: gfx94X-dcgpu }
+          - { AMDGPU_FAMILIES: gfx110X-dgpu }
+          - { AMDGPU_FAMILIES: gfx120X-all }
```

Comment:
Weren't we going to reuse the matrix from https://github.com/ROCm/TheRock/blob/main/build_tools/github_actions/amdgpu_family_matrix.py and other workflows? Why is gfx1151 missing from this list?